src/main/java/reposense/system/CommandRunner.java

```diff
@@ -108,6 +112,24 @@ public static String getCommitHashBeforeDate(String root, String branchName, Dat
         return runCommand(rootPath, revListCommand);
     }
 
+    /**
+     * Removes the author for the {@code commitHash} in the {@code branch} of the repo.
+     */
+    public static void removeCommitAuthor(String root, String branch, String commitHash) {
+        Path rootPath = Paths.get(root);
+        // create temp branch with the given commitHash as the HEAD
+        runCommand(rootPath, String.format(CHECKOUT_COMMIT_IN_TEMP_BRANCH_COMMAND_FORMAT, commitHash));
```

Comment:
note: this command will fail if a `temp` branch already exists. we should attempt to delete `temp` branch first or use a unique branch name

Reply:
Hmmm, your right, perhaps we can use the `commitHash` as the branch name instead, highly doubt any1 would put their branch names as that xD